Presentation.Web/Presentation.Web.csproj

```diff
@@ -344,6 +344,7 @@
   </ItemGroup>
   <ItemGroup>
     <TypeScriptCompile Include="app\components\local-config\ui-customization\local-ui-customization.directive.ts" />
+    <TypeScriptCompile Include="app\helpers\FormatHelper.ts" />
```

Comment:
remember to apply the naming convention... `select2-option-format-helper.ts` (notice how I added some more context to the name)